docs/building-modules/structure.md

```diff
@@ -25,6 +25,7 @@ x/{module}
 │   ├── invariants.go
 │   ├── genesis.go
 │   ├── keeper.go
+│   ├── msg_server.go
 │   ├── ...
 │   └── querier.go
```

Comment:
querier.go is gone btw

Reply:
well it's still there for legacy queriers, but yeah maybe better to drop it. In this regard, maybe `handler.go` should be dropped as well?

Reply:
Ah yeah, true. Let's maybe keep it in this case, since it's the only place we talk about legacy stuff.